sklearn/metrics/regression.py

```diff
@@ -33,7 +36,7 @@
 ]
 
 
-def _check_reg_targets(y_true, y_pred):
+def _check_reg_targets(y_true, y_pred, output_weights):
```

Comment:
output_weights=None isn't a sane default: it returns a whole array. 'uniform' is better as default. In any case I'll make it explicit to correct median_absolute_error.